volume-checker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.16-alpine
+
+WORKDIR /app
+
+COPY *.go ./
+
+RUN go env -w GO111MODULE=off
+
+RUN go build -o /volume-checker
+
+RUN mkdir /volume
+
+RUN touch /volume/hello.world
+
+EXPOSE 8080
```

Comment:
Is `EXPOSE` needed? If deployed to Cloud Run this is set at runtime

Reply:
No, removed